cmd/geth/usage.go

```diff
@@ -233,6 +236,13 @@ var AppHelpFlagGroups = []flagGroup{
 		Name:  "WHISPER (EXPERIMENTAL)",
 		Flags: whisperFlags,
 	},
+	{
```

Comment:
Moving the command line argument `ISTANBUL` section to be above the `DEPRECATED` and `MISC` sections when `--help` is used.